### ISSUE TITLE
Social Media Best Practices

### ISSUE THREAD

**Issue by schaekat** [smallbusinesshero/business#31]
i think you said you might have some slides lying around that could be useful, right?

**Comment by jesshurf** [comment]
Yes! I'm working on refreshing it to be more tuned to what we need for SBH. Should be ready to send tomorrow.

**Comment by jesshurf** [comment]
This document is saved in the Marketing Google Drive folder in SBH: [https://docs.google.com/document/d/13k72yV_pWBFutRrgILkuM9B4jRmHAgjuR5PnfQA-cgU/edit?usp=sharing](url)
